NDTensors/src/Unwrap/src/expose.jl

```diff
@@ -0,0 +1,9 @@
+struct Exposed{Unwraped,Object}
+  object::Object
+end
+
+expose(object) = Exposed{unwrap_type(object),typeof(object)}(object)
+
+unexpose(E::Exposed) = E.object
+
+Base.parent(E::Exposed) = parent(unexpose(E))
```

Comment:
I think this should be moved into a file in `Unwrap/src/functions/`.

Reply:
Okay! Makes sense I can define some of the other small functions like this in the same file